build-tools/update-copyright.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Description: This script recursively searches the folder starting at FILEPATH
+# for all Copyright notices and both formats and updates them to extend from the
+# first year listed to the year provided by the user
+
+# Defines relative path from this file to top of directory where a recursive
+# copyright update will be performed
+FILEPATH="../"
+
+printf "\nAll instances of Copyright notices:\n":
+# Displays all Copyright lines excluding those in the .git folder
+grep -r -n --exclude-dir=".git" --exclude="update-copyright.sh" "Copyright (*[cC]*)* *20" $FILEPATH | grep "F5 Networks"
```

Comment:
Find a way to store this as a var then add that | wc line to that var for efficiency.